src/include/ifl_internal.h

```diff
@@ -45,10 +45,14 @@ extern "C" {
 
 #include "pbs_ifl.h"
 
+void __attribute__((constructor)) set_ifl_funcs(void);
```

Comment:
This broke on windows since it is gcc specific syntax.  Is there a reason we can't just initialize the global pointers directly when defined?

Reply:
One of the things I've been doing with my PMIx prototype work is to define all of the function pointers and set them to NULL initially. I then define macros for each one that test whether the function pointer is NULL and call the function if not. This is all based on runtime loading using dlopen() so that if some functions aren't defined, the program will still use the functions that are mapped. That's probably not strictly necessary here, but it is an option.

Reply:
Thanks for the suggestions guys. Mike, using dlopen() won't work with static linking (right?), which might be an unnecessary constraint. The simpler solution should suffice here, so I think I'll go with that.